src/shared/components/Header/Menu/MenuListItem/component.tsx

```diff
@@ -0,0 +1,42 @@
+import React, { useState } from 'react';
+
+import { MenuList, MenuItem } from 'shared/types';
+import { ArrowIcon } from 'shared/icons';
+
+import { ArrowIconWrapper, ListWrapper } from './styles';
+
+type MenuListItem = {
+  list: MenuList;
+};
+
+const MenuListItem: React.FC<MenuListItem> = ({ list }: MenuListItem) => {
+  const [isOpenList, setIsOpenList] = useState(false);
+
+  const onToggleHandler = (): void => {
+    setIsOpenList(!isOpenList);
+  };
+
+  return (
+    <nav>
+      <strong
+        onClick={(): void => {
+          onToggleHandler();
```

Comment:
Wh do you need one more function, but not ` onClick={() => setIsOpenList(!isOpenList);}`?

Reply:
Done!